README.md

```diff
@@ -15,11 +15,14 @@ See [Usage](README.md#Usage) for more details.
 
 ## Requirements
 
-- JupyterLab 3
+- JupyterLab >= 4
```

Comment:
With JupyterLab 5 surely coming, I think it is better to drop the `>=` here :smile:  
```suggestion
- JupyterLab 4
```

Reply:
Fair enough, I agree!!